quelpa.el

```diff
@@ -235,9 +244,7 @@ On error return nil."
              (and pkg-desc
                   (version-list-<=
                    (version-to-list version)
-                   (if (functionp 'package-desc-vers)
-                       (package-desc-vers pkg-desc) ;old implementation
-                     (package-desc-version (car pkg-desc))))))
+                   (package-desc-version (car pkg-desc)))))
```

Comment:
You mentioned earlier maintaining support for Emacs 24.3.  Does this code you removed go back further than that?  Just curious.

Reply:
`package-desc-ver`v is already deprecated in 24.3, so we can start using new function `package-desc-version` from there. Keeping void function around is not good IMO.